### PR TITLE
AFA walkthrough fixes

### PR DESCRIPTION
**Pull request by mattandrews**
A variety of copy/UX changes – see [the doc](https://docs.google.com/document/d/1OSq5yW8EMEuaBXLD4HBYKXwbOR-OYBHIXcqz5t3W3Hg/edit#) for a rundown.

### CODE REVIEW

File: common/content-api.js [hunk 1]
```diff
@@ -72,7 +72,9 @@ function mergeWelshBy(propName) {
             return enResults;
```
[the remark this answers is not in the document]
Comment (mattandrews) [reply common/content-api.js]: Yeah, couldn't get it to not change these files even when I hadn't modified them...?!

[the remark this answers is not in the document]
Comment (mattandrews) [reply common/content-api.js]: (reverting them just made it happen again...?!?)

File: controllers/apply/form-router-next/views/summary.njk [hunk 2]
```diff
@@ -7,6 +7,9 @@
 
 {% block title %}{{ title }} | {{ formTitle }} | {% endblock %}
 
+{% set formIsComplete = formProgress.isComplete %}
+{% set formIsPristine = formProgress.isPristine %}
```
[the remark this answers is not in the document]
Comment (mattandrews) [reply controllers/apply/form-router-next/views/summary.njk]: Yeah, they stop the getter from executing every time we use these variables (eg. and looping over the entire form object each time)

File: controllers/apply/form-router-next/eligibility.js [hunk 3]
```diff
@@ -2,11 +2,17 @@
...
 
-module.exports = function(eligibilityBuilder) {
+const commonLogger = require('../../../common/logger');
+
+module.exports = function(eligibilityBuilder, formId) {
     const router = express.Router();
 
     const templatePath = path.resolve(__dirname, './views/eligibility');
 
+    const logger = commonLogger.child({
```
[the remark this answers is not in the document]
Comment (mattandrews) [reply controllers/apply/form-router-next/eligibility.js]: Done!

File: common/content-api.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (davidrapson) [reply common/content-api.js]: Do you have some kind of file watcher set up in your editor? Not a big deal though.

**Pull request merged**